thorlcr/activities/countproject/thcountprojectslave.cpp

```diff
@@ -53,14 +53,14 @@ class LocalCountProjectActivity : public BaseCountProjectActivity
     }
     virtual void start()
     {
-        ActivityTimer s(totalCycles, timeActivities);
+        ActivityTimer s(slaveTimerStats, timeActivities);
```

Comment:
same goes here - it is hard to review because most of the changes are simply a rename,